index.html

```diff
@@ -7,11 +7,13 @@
   </head>
   <body>
     <h1>Which product do you want the most?</h1>
-    <section>
+    <section id="pic-container">
       <img src="" alt="" id="image-one">
       <img src="" alt="" id="image-two">
       <img src="" alt="" id="image-three">
     </section>
+    <ul id="something">
```

Comment:
oh man, I didn't realize that we left this id as 'something'.  you should probably change this to a more meaningful name.